src/bin/server.ts

```diff
@@ -6,6 +6,7 @@ import { Driver } from "zwave-js";
 import { ZwavejsServer } from "../lib/server";
 import { createMockDriver } from "../mock";
 import { parseArgs } from "../util/parse-args";
+import { normalizeKey } from "../util/normalize-key";
```

Comment:
Functions that are only used in 1 file can just be put in the same file, no need to create their own file.